osu.Game/Utils/ColourUtils.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Game.Graphics;
+using osuTK.Graphics;
+
+namespace osu.Game.Utils
+{
+    public static class ColourUtils
+    {
+        /// <summary>
+        /// Returns a foreground text colour that is supposed to contrast well on top of
+        /// the supplied <paramref name="backgroundColour"/>.
+        /// </summary>
+        public static Color4 ForegroundTextColourFor(Color4 backgroundColour)
```

Comment:
Not a huge deal, buy any reason for not putting this function inside `OsuColour` itself (since that seems to be where we have all other `static` colour functions, and this function itself calls that class anyway)?

Reply:
No good reason. I'll move it right now.